lib/scope.js

```diff
@@ -2741,9 +2746,8 @@ module.exports = {
 
       // If this is a custom datatype
       let custom_datatype = await scope.get_custom_datatype( scope, { handle });
-      let is_custom_datatype = scope.get_random_input_for[ custom_datatype ] !== undefined;
+      let is_custom_datatype = scope.get_random_input_for[ custom_datatype ] !== undefined && type != 'radio';
       if ( is_custom_datatype ) {
```

Comment:
```suggestion
      let is_custom_datatype = scope.get_random_input_for[ custom_datatype ] !== undefined;
      if ( is_custom_datatype && type !== `radio` ) {
```

Otherwise, the name `is_custom_datatype` doesn't mean what you'd expect it to mean. I'm not sure why this excludes custom datatypes that are radio buttons. Where do we deal with custom datatypes that are radio buttons?

Reply:
First, I don't think we have any custom datatypes that are radio buttons.

Second, this probably shouldn't need to exist, I just know that `get_custom_datatype` and `is_custom_datatype` would say that radio buttons were custom datatypes when they weren't actually. I would guess that `get_custom_datatype` isn't correct, but I don't know how to fix it without breaking other things.

Reply:
Made #807